src/pages/SupplyRequisitionPage.js

```diff
@@ -0,0 +1,362 @@
+    });
+  }
+
+  onUseSuggestedQuantities() {
+    this.props.runWithLoadingIndicator(() => {
+      const { database, requisition } = this.props;
+      database.write(() => {
+        requisition.setRequestedToSuggested(database);
+        database.save('Requisition', requisition);
+      });
+      this.refreshData();
+    });
+  }
+
+  onSelectionChange(newSelection) {
+    this.setState({ selection: newSelection });
+  }
+
+  getModalTitle() {
+    const { ITEM_SELECT, COMMENT_EDIT } = MODAL_KEYS;
+    switch (this.state.modalKey) {
+      default:
+      case ITEM_SELECT:
+        return modalStrings.search_for_an_item_to_add;
+      case COMMENT_EDIT:
+        return modalStrings.edit_the_requisition_comment;
+    }
+  }
+
+  updateDataFilters(newSearchTerm, newSortBy, newIsAscending) {
+    // We use != null, which checks for both null or undefined (undefined coerces to null)
+    if (newSearchTerm != null) this.dataFilters.searchTerm = newSearchTerm;
+    if (newSortBy != null) this.dataFilters.sortBy = newSortBy;
+    if (newIsAscending != null) this.dataFilters.isAscending = newIsAscending;
+  }
+
+  /**
+   * Returns updated data according to searchTerm, sortBy and isAscending.
+   */
+  refreshData(newSearchTerm, newSortBy, newIsAscending) {
+    this.updateDataFilters(newSearchTerm, newSortBy, newIsAscending);
+    const { searchTerm, sortBy, isAscending } = this.dataFilters;
+    const data = this.props.requisition.items
+                 .filtered('item.name BEGINSWITH[c] $0 OR item.code BEGINSWITH[c] $0', searchTerm);
+    let sortDataType;
+    switch (sortBy) {
+      case 'itemCode':
+      case 'itemName':
+        sortDataType = 'string';
+        break;
+      case 'monthlyUsage':
+      case 'suggestedQuantity':
+      case 'requiredQuantity':
+        sortDataType = 'number';
+        break;
+      default:
+        sortDataType = 'realm';
+    }
+    this.setState({ data: sortDataBy(data, sortBy, sortDataType, isAscending) });
+  }
+
+  openModal(key) {
+    this.setState({ modalKey: key, modalIsOpen: true });
+  }
+
+  closeModal() {
+    this.setState({ modalIsOpen: false });
+  }
+
+  openItemSelector() {
+    this.openModal(MODAL_KEYS.ITEM_SELECT);
+  }
+
+  openCommentEditor() {
+    this.openModal(MODAL_KEYS.COMMENT_EDIT);
+  }
+
+  renderPageInfo() {
+    const { requisition } = this.props;
+    const infoColumns = [
+      [
+        {},
+        {
+          title: `${pageInfoStrings.entry_date}:`,
+          info: formatDate(requisition.entryDate),
+        },
+        {
+          title: `${pageInfoStrings.entered_by}:`,
+          info: requisition.enteredByName,
+        },
+      ],
+      [
+        {
+          title: `${pageInfoStrings.supplying_store_name}`,
+          info: requisition.otherStoreName.name,
+        },
+        {
+          title: `${pageInfoStrings.months_stock_required}:`,
+          info: Math.round(requisition.monthsToSupply),
```

Comment:
Umm I copy pasted from RequisitionPage, ( I don't see any use for Math.round.. ).. changed in both.